### PR TITLE
feat(email-templates) - certification email templates

### PR DESCRIPTION
**Pull request by kalidiagne**
# Description

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by kalidiagne** [comment]
@julien51 can you please review the copy?

**Review by searchableguy — APPROVED** [review]
I think we need some text changes but code wise LGTM

File: packages/email-templates/src/templates/certificationKeyAirdropped.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import handlebars from 'handlebars'
+import { customContentStyle } from './helpers/customContentStyle'
+import { links } from './helpers/links'
+
+handlebars.registerHelper('links', links)
+
+export default {
+  subject: `Your certification for {{{lockName}}}`,
+  html: `<h1>A NFT certification for "{{lockName}}" was airdropped to you!</h1>
+<p><a href="{{certificationUrl}}">Here</a> is your NFT certification (#{{keyId}}) for <strong>{{lockName}}</strong> was just airdropped!</p>
```
Comment (julien51) [inline packages/email-templates/src/templates/certificationKeyAirdropped.ts:10]: Eventually we can make these as PDF that are attached to the email

Comment (julien51) [reply packages/email-templates/src/templates/certificationKeyAirdropped.ts:10]:
```suggestion
<p><a href="{{certificationUrl}}">Here</a> is your NFT certification (#{{keyId}}) for <strong>{{lockName}}</strong>. It was also just airdropped to your address.</p>
```

If the certification was airdropped by email we need to show the link to claim... please add that!

File: packages/email-templates/src/templates/certificationKeyAirdropped.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+{{#if customContent}}
+<section style="${customContentStyle}">
+{{{customContent}}}
+  </section>
+{{/if}}
+
+<p>It has been added to your <a href="{{keychainUrl}}">Unlock Keychain</a>, where you can view it and, if needed, print it as a signed QR Code!</p>
```
Comment (julien51) [inline packages/email-templates/src/templates/certificationKeyAirdropped.ts:18]: Only show that if it was indeed airdropped to the user's wallet... and if not show the link to claim it.

File: packages/email-templates/src/templates/certificationKeyMined.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import handlebars from 'handlebars'
+import { customContentStyle } from './helpers/customContentStyle'
+import { links } from './helpers/links'
+
+handlebars.registerHelper('links', links)
+
+export default {
+  subject: `Your certification for {{{lockName}}}`,
+  html: `<h1>A NFT certification for "{{lockName}}" was added to your wallet!</h1>
+<p><a href="{{certificationUrl}}">Here</a> is your NFT certification (#{{keyId}}) for <strong>{{lockName}}</strong> was just minted!</p>
```
Comment (julien51) [inline packages/email-templates/src/templates/certificationKeyMined.ts:10]:
```suggestion
  html: `<h1>A NFT certification for "{{lockName}}" was added to your wallet!</h1>
<p>Your NFT certification (#{{keyId}}) for <strong>{{lockName}}</strong> was just mined. You can <a href="{{certificationUrl}}">consult it directly</a> on this web page.</p>
```

File: packages/email-templates/src/templates/certificationKeyMined.ts [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+{{#if customContent}}
+<section style="${customContentStyle}">
+{{{customContent}}}
+  </section>
+{{/if}}
+
+<p>It has been added to your <a href="{{keychainUrl}}">Unlock Keychain</a>, where you can view it and, if needed, print it as a signed QR Code!</p>
```
Comment (julien51) [inline packages/email-templates/src/templates/certificationKeyMined.ts:18]:
```suggestion
<p>It has been added to your <a href="{{keychainUrl}}">Unlock Keychain</a>, where you can view it and its metadata.</p>
```

**Review by julien51 — CHANGES_REQUESTED** [review]
Please fix the copy and add the link to claim.

File: packages/email-templates/src/templates/certificationKeyAirdropped.ts [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+<p><a href="{{certificationUrl}}">Here</a> is your NFT certification (#{{keyId}}) for <strong>{{lockName}}</strong>. It was also just airdropped to your address.</p>
+
+{{#if customContent}}
+<section style="${customContentStyle}">
+{{{customContent}}}
+  </section>
+{{/if}}
+
+{{#if isUserAddress}}
+  <p>It has been added to your <a href="{{keychainUrl}}">Unlock Keychain</a>, where you can view it and, if needed, print it as a signed QR Code!</p>
```
Comment (julien51) [inline packages/email-templates/src/templates/certificationKeyAirdropped.ts:19]:
```suggestion
  <p>It has also been added to your <a href="{{keychainUrl}}">Unlock Keychain</a>, where you can view it and, if needed, print it as a signed QR Code!</p>
```

File: packages/email-templates/src/templates/certificationKeyAirdropped.ts [hunk 6]
```diff
@@ -0,0 +1,25 @@
+import handlebars from 'handlebars'
+import { customContentStyle } from './helpers/customContentStyle'
+import { links } from './helpers/links'
+
+handlebars.registerHelper('links', links)
+
+export default {
+  subject: `Your certification for {{{lockName}}}`,
+  html: `<h1>A NFT certification for "{{lockName}}" was airdropped to you!</h1>
+<p><a href="{{certificationUrl}}">Here</a> is your NFT certification (#{{keyId}}) for <strong>{{lockName}}</strong>. It was also just airdropped to your address.</p>
```
Comment (julien51) [inline packages/email-templates/src/templates/certificationKeyAirdropped.ts:10]:
```suggestion
<p><a href="{{certificationUrl}}">Here</a> is your NFT certification (#{{keyId}}) for <strong>{{lockName}}</strong>. It was just airdropped to your address.</p>
```

**Review by julien51 — APPROVED** [review]

**Pull request merged**